### PR TITLE
src/Cedar/Session: resolve coverity "issue", remove unused functions

### PR DESCRIPTION
**Pull request by chipitsine**
SoftEther VPN Patch Acceptance Policy:
http://www.softether.org/5-download/src/9.patch

You have two options which are described on the above policy.
Could you please choose either option 1 or 2, and specify it clearly on the reply?

- one

### CODE REVIEW

File: src/Cedar/Session.c [hunk 1]
```diff
@@ -1261,21 +1261,10 @@ void StopSessionEx(SESSION *s, bool no_wait)
...
-	if (s->ServerMode == false)
-	{
-		// Client mode
-		if (s->Connection)
-		{
-			StopConnection(s->Connection, no_wait);
-		}
-	}
-	else
+	// both Server and Client modes
```
Comment (davidebeatrici) [inline src/Cedar/Session.c]: I would change the comment to:
```
// Server and client mode
```

Comment (chipitsine) [reply src/Cedar/Session.c]: "modes" ?

Comment (davidebeatrici) [reply src/Cedar/Session.c]: I think that `modes` makes it sound like as if there were multiple client/server modes.

**Comment by chipitsine** [comment]
accidently superceeded by https://github.com/SoftEtherVPN/SoftEtherVPN/pull/682

**Pull request closed without merging**